include/debug.h

```diff
@@ -42,6 +45,9 @@ noreturn void panic(const char *fmt, ...) __printf(1, 2);
 #define error(...) log(LOG_STRING_ERROR __VA_ARGS__)
 #define warn(...)  log(LOG_STRING_WARNING __VA_ARGS__)
 #define info(...)  log(LOG_STRING_INFO __VA_ARGS__)
+#if TEST
```

Comment:
Same here. Please put this after `DEBUG`.

Reply:
done